be/src/vec/runtime/vfile_result_writer.cpp

```diff
@@ -327,7 +325,7 @@ Status VFileResultWriter::_write_csv_file(const Block& block) {
                     break;
                 }
                 default: {
-                    // not supported type, like BITMAP, HLL, just export null
+                    // not supported type, like BITMAP, OBJECT, HLL, just export null
```

Comment:
I think we should use SessionVariable `return_object_data_as_binary` to control whether output HLL/BitMap as ParquetWriter did.

Reply:
done